### DISCUSSION TITLE
Drag and drop - can we define multiple correct destinations?

### DISCUSSION THREAD

**Discussion by corinnewo** [ji-devs/ji-cloud#1561] [General]
@dakom Can we define two different destinations for the drag and drop - eg if 2 alefs in the word, then it should not matter which alef goes where. Both could be correct.

**Comment by dakom** [comment]
I'd say the starting point would be how to allow this in the editor

**Comment by corinnewo** [comment]
I've added to the ideation agenda - to discuss how to do it
